### PR TITLE
[FEATURE] Display Plugins in "New Content Element" Wizard

### PR DESCRIPTION
**Pull request by linawolf**
resolves https://github.com/TYPO3-Documentation/tea/issues/825

### CODE REVIEW

File: Configuration/Icons.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    // Icon identifier
+    'tea_icon' => [
+        // Icon provider class
+        'provider' => \TYPO3\CMS\Core\Imaging\IconProvider\SvgIconProvider::class,
+        // The source SVG for the SvgIconProvider
+        'source' => 'EXT:tea/Resources/Public/Icons/Extension.svg',
```
Comment (oliverklee) [inline Configuration/Icons.php:9]: Please let's use a dedicated file for this (even if we copy the extension icon). My proposal for a name is `Plugin.svg`.

File: Configuration/TsConfig/Page/newContentElement.tsconfig [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        description = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index.description
+        iconIdentifier = tea_icon
+        tt_content_defValues {
+          CType = list
+          list_type = = tea_teaindex
+        }
+      }
+      tea_teashow {
+        title = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_show
+        description = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_teashow.description
```
Comment (oliverklee) [inline Configuration/TsConfig/Page/newContentElement.tsconfig:16]: tea_teashow -> tea_show

File: Configuration/TsConfig/Page/newContentElement.tsconfig [hunk 3]
```diff
@@ -0,0 +1,27 @@
+mod.wizards.newContentElement.wizardItems {
+  // add the plugins to the "Create new content element" wizard
+  plugins {
+    elements {
+      tea_teaindex {
+        title = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index
+        description = LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea_index.description
```
Comment (oliverklee) [inline Configuration/TsConfig/Page/newContentElement.tsconfig:7]: We also need to add both descriptions to the EN and DE language files.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: Configuration/Icons.php (hunk 1 above)
Comment (linawolf) [reply Configuration/Icons.php]: Done, i also used the same one for the plugin registration

File: Configuration/TsConfig/Page/newContentElement.tsconfig (hunk 3 above)
Comment (linawolf) [reply Configuration/TsConfig/Page/newContentElement.tsconfig:7]: Done

**Review by oliverklee — CHANGES_REQUESTED** [review]
I've tested this on V11. Both for the list view and single view created with this wizard, I see "no plugin selected" for the content element in the BE.

Also, the UI for the list and single view content elements is not consistent: The list view item does not have the term "view" in the title, but it has a description, while the single view has the term "view" in the title but has no description:

image

I propose to use the title "Tea list view" for the list view, and add a description the single view.

**Comment by oliverklee** [comment]
@linawolf Would you like to continue work on this PR, or would you prefer someone took it over?

**Comment by linawolf** [comment]
Sorry, kind of forgot this issue, will put it on my list again

**Comment by lukaszuznanski** [comment]
Hi,

We've implemented a policy to close pull requests that have been inactive for over six months. If you're still interested in contributing and would like to continue working on this pull request, we're more than happy to keep it open.

Please take a moment to review your pull request and let us know if you'd like to proceed with it. If we don't receive a response within the next 30 days, we'll assume that you no longer wish to pursue it, and we'll proceed with closing the pull request.

Feel free to reach out if you have any questions or need further assistance.

**Pull request closed without merging**